xml/libvirt_configuration_virsh.xml

```diff
@@ -569,7 +569,32 @@ CPU Affinity:   yy
     See <link xlink:href="https://libvirt.org/pci-hotplug.html"/> for more
     information about PCI topology between &qemu; machine types.
     </para>
-  </important>
+   </important>
+
+   <sect2 xml:id="tip-libvirt-config-zpci">
+    <title>&pciback; for &zseries;</title>
+    <para>
+     &qemu; can configure zPCI attributes that are used when configuring &pciback;
+     for the &zseries; architecture. <option>uid</option> represents user-defined
+     identifier, while <option>fid</option> represents PCI function identifier.
+     These attributes are optional and if you do not specify them, they are
+     automatically generated with non-conflicting values.
+    </para>
+    <para>
+     These attributes are implemented by using a new PCI address extension flag.
+     To include zPCI attribute in your domain specification, use the following
+     example definition:
+    </para>
+<screen>
+&lt;hostdev mode='subsystem' type='pci'>
```

Comment:
Need `&gt;` here and a few places below, right?

Reply:
the closing bracket is not needed in fact :-)